### ISSUE TITLE
Very high RAM usage with large marker sizes

### ISSUE THREAD

**Issue by issacdowling** [flxzt/rnote#664]
**Describe the bug**  
If I use excessively large sizes of pen, RAM usage goes very high (many gigs), which then leads to an OOM crash.

**To Reproduce**  
Steps to reproduce the behaviour:
1. Select the marker
2. Set the size to 200 (applies to lower sizes in the 100s too)
3. Scribble, watch RAM usage skyrocket

**Expected behavior**  
I would expect less RAM usage. I would be fine if it went to swap or something, however consistently it just crashes with huge marker sixes.

**Console Output**  
```
 2023-05-10T10:09:23.250Z DEBUG rnote > ... env_logger initialized
Terminated
```

**Screenshots**  
![Screenshot from 2023-05-10 11-11-24](https://github.com/flxzt/rnote/assets/133106903/ed01de0a-7fba-484d-9389-951d72041278)
![Screenshot from 2023-05-10 11-18-49](https://github.com/flxzt/rnote/assets/133106903/ccd88369-31ae-43c9-8fae-4d27161a7672)

**Desktop (please complete the following information):**  
 - OS: Fedora 38
 - App Version: 0.6.0
 - Installation Source: Flatpak
 - Desktop Environment: Gnome 44

**Additional context**  
I understand that using such large pen sizes is silly, and this isn't an urgent (or really necessary) bug to fix, but I thought I should report it anyway. Rnote is awesome.

**Comment by Vertecedoc4545** [comment]
here too, but is gradual, regardless of the stroke size, but when you are in biger strokes is a lot fater,  
![Skjermbilde fra 2023-06-01 01-23-18](https://github.com/flxzt/rnote/assets/58826034/bb19332a-fa86-4ba0-8aa4-89187c6146bc)

**Comment by Vertecedoc4545** [comment]
how are saved in memory the strokes, how are them prossed??

**Comment by flxzt** [comment]
This should be fixed in v0.7.0, strokes with large widths are now rendered as a single image which should reduce the memory usage significantly.
